DATA/EQUIPMENT/market_misc.ini

```diff
@@ -15079,6 +15306,30 @@ MarketGood = li_shield_graviton_light_adv01, 0, 0, 10, 10, 0, 1
 MarketGood = rh_shield_molecular_light_adv01, 0, 0, 10, 10, 0, 1
 MarketGood = ku_shield_neutrino_light_adv01, 0, 0, 10, 10, 0, 1
 MarketGood = br_shield_positron_light_adv01, 0, 0, 10, 10, 0, 1
+MarketGood = li_shield_graviton_heavy_opt01, 0, 0.2, 10, 10, 0, 1

```

Comment:
Do we really want to sell technology of every house at the Order for even lower reputation? Nomads are not needing _that_ many shots anymore to get Order up - even if it still stays a challenge.

Reply:
The House shields become the new base line of shields 

Standatd = Civilian (entry level)

Advanced & Optimized (Basic House Equipment)

Even the pirates sell these basic "house shields" because they stole the technology and use it for themselves, the order has access to all of them and getting Order Rep is far more difficult due to nomads are a threat, long travel time, it needs a few kills to get order at least 6 bars.

Its just defensive equipment so i personaly see no problem here to sell all basic house shields in here. 

But selling Codenames of houses, you are right it will be cut out of the market entry